### ISSUE TITLE
Missing Null Safety Support!

### ISSUE THREAD

**Issue by Raunakk02** [aeyrium/bezier-chart#116]
This package does not support sound null safety. It needs to be migrated to a null-safe version.

For reference on migrating to null safety visit [https://dart.dev/null-safety/migration-guide#switch-to-the-dart-212-release](URL)

**Comment by calvintam236** [comment]
duplicate #111

**Comment by Raunakk02** [comment]
Should I close this issue? @calvintam236 can you tell me when I can expect the null safety to be available on pub.dev?

**Comment by calvintam236** [comment]
> Should I close this issue? @calvintam236 can you tell me when I can expect the null safety to be available on pub.dev?

@Raunakk02 for this you have to ask @diegoveloper since he is the only one has access on pub.dev.

**Comment by diegoveloper** [comment]
Yeah, sorry I've been so busy these months, I tried to publish the package but it's in stuck -> https://github.com/aeyrium/bezier-chart/actions

**Comment by UNIcodeX** [comment]
hmm. says `Unexpected input(s) 'version_name', valid inputs are ['file']`

**Comment by diegoveloper** [comment]
> hmm. says `Unexpected input(s) 'version_name', valid inputs are ['file']`

that's just a warning -> https://github.com/aeyrium/bezier-chart/runs/3107136431?check_suite_focus=true

**Comment by calvintam236** [comment]
@diegoveloper looks like it is asking you to do oauth authorization per the log. Maybe there're some config changes?

**Comment by calvintam236** [comment]
https://github.com/aeyrium/bezier-chart/blob/27d3175f0556a46417a9ed0225ea91f09077ecab/.github/workflows/release.yml#L61

I think your credential has expired and need to be updated.

@diegoveloper Please update it and try trigger the action again. Thanks!

**Comment by diegoveloper** [comment]
Yeah, I'm waiting for the admin response :/

**Comment by calvintam236** [comment]
@diegoveloper Any updates? Can you also release a new tag so we can use it in local?

**Comment by calvintam236** [comment]
> Yeah, I'm waiting for the admin response :/

@diegoveloper Any updates?

**Comment by diegoveloper** [comment]
no :/ it would be better if you fork the project
